drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -45,7 +45,8 @@ function DragNDropTemplates(url_name) {
             'draggable': !item.drag_disabled,
             'aria-grabbed': item.grabbed,
             'data-value': item.value,
-            'data-drag-disabled': item.drag_disabled
+            'data-drag-disabled': item.drag_disabled,
+            'aria-describedby': gettext('Press "Enter", "Space", "Ctrl-m", or "⌘-m" on an item to select it for dropping, then navigate to the zone you want to drop it on.'),
```

Comment:
@cptvitamin So currently there is a describedby for when the item is correctly placed. So then I assume we'd want the default describedby to be present in all other cases, right? Doing that works as long as I change a few lines in assert_reverted_item() in tests_integration.py.